arcgis-ios-sdk-samples/Maps/Set max extent/SetMaxExtentViewController.swift

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class SetMaxExtentViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // The graphics overlay used to show the extent envelope graphic.
+            let graphicsOverlay = AGSGraphicsOverlay()
+            graphicsOverlay.graphics.add(AGSGraphic(
+                geometry: extentEnvelope,
+                symbol: AGSSimpleLineSymbol(style: .dash, color: .red, width: 5)
+            ))
+            mapView.graphicsOverlays.add(graphicsOverlay)
+        }
+    }
+    
+    // MARK: Properties
+    
+    /// The envelope that represents the max extent.
+    let extentEnvelope: AGSEnvelope = {
+        let northwestPoint = AGSPointMakeWebMercator(-12139393.2109, 5012444.0468)
+        let southeastPoint = AGSPointMakeWebMercator(-11359277.5124, 4438148.7816)
+        return AGSEnvelope(min: northwestPoint, max: southeastPoint)
+    }()
```

Comment:
Please don't use any of the `AGSPointMake` functions; they're really a convenience for Objective-C. In Swift, one of the initializers should be used instead, similar to how we don't use `CGPointMake` in Swift. You could use `AGSPoint.init(x:y:spatialReference:)`:

```swift
let extentEnvelope: AGSEnvelope = {
    let northwestPoint = AGSPoint(x: -12139393.2109, y: 5012444.0468, spatialReference: .webMercator())
    let southeastPoint = AGSPoint(x: -11359277.5124, y: 4438148.7816, spatialReference: .webMercator())
    return AGSEnvelope(min: northwestPoint, max: southeastPoint)
}()
```

But it would be simpler to use a different initializer on `AGSEnvelope`.

```suggestion
    let extentEnvelope = AGSEnvelope(
        xMin: -12139393.2109,
        yMin: 4438148.7816,
        xMax: -11359277.5124,
        yMax: 5012444.0468,
        spatialReference: .webMercator()
    )
```

Reply:
Will fix in #1169